pages/technology_categories/technology_categories.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: sideNavigation
+title: Technology Categories
+permalink: /technology-categories
+redirect_from:
+    - /information
+---
+
+# Technology Categories 
```

Comment:
Probably need some better content for this page.

Reply:
Given no previous page for information, seem a basic list of sub page will suffice. Can add more at later date if needed